docs/get-started/getting-started-workload-qualification.md

```diff
@@ -124,7 +124,7 @@ the driver logs with `spark.rapids.sql.explain=all`.
    this version:
    
    ```
-   !NOT_FOUND <RowDataSourceScanExec> cannot run on GPU because no GPU enabled version of operator class org.apache.spark.sql.execution.RowDataSourceScanExec could be found
+   !Unsupported operator: <RowDataSourceScanExec> cannot run on GPU because no GPU enabled version of operator class org.apache.spark.sql.execution.RowDataSourceScanExec could be found
```

Comment:
This still says "no GPU enabled version of operator class .... could be found".  The original problem that triggered #4088 was that the "class not found" message led the user to believe this was a classloader/classpath issue rather than a normal situation where we know this Catalyst operation is not supported.